packages/react/collapsible/src/Collapsible.tsx

```diff
@@ -201,6 +208,8 @@ const CollapsibleContentImpl = React.forwardRef<
       style={{
         [`--radix-collapsible-content-height` as any]: height ? `${height}px` : undefined,
         [`--radix-collapsible-content-width` as any]: width ? `${width}px` : undefined,
+        // We prevent consumer animation if open when mounted to prevent layout from moving around
```

Comment:
```suggestion
        // Prevent animations running when initially mounted in the open state
        // this avoids unexpected layout shifts
```

Is that any better? 😅

Reply:
perfect 👌

Reply:
Tweaked it a little (shorter) but it's the same gist 😛

Reply:
Best of both 😄